internal/cmd/network/network.go

```diff
@@ -14,22 +15,29 @@ func NewCommand(s state.State) *cobra.Command {
 		TraverseChildren:      true,
 		DisableFlagsInUseLine: true,
 	}
+	cmd.AddGroup(&cobra.Group{ID: "general", Title: "General"})
+	cmd.AddGroup(&cobra.Group{ID: "protection", Title: "Protection"})
+	cmd.AddGroup(&cobra.Group{ID: "route", Title: "Routes"})
+	cmd.AddGroup(&cobra.Group{ID: "subnet", Title: "Subnets"})
+
 	cmd.AddCommand(
```

Comment:
```suggestion
	cmd.AddCommand(
		util.WithGroup("general", CreateCmd.CobraCommand(s)),
```

`create` is missing from the new list

Reply:
Oops, seems like I removed create for one resource (likely Image?) and then copied it over without adding it back. Maybe we should change the tests so that they run on the root command instead of instantiating the subcommands, that would catch these kinds of mistakes.

Reply:
We could also make a snapshot of the help outputs to make sure they are what we expect.